tests/WorkflowJob.py

```diff
@@ -0,0 +1,74 @@
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2020 Steve Kowalik <steven@wedontsleep.org>                        #
```

Comment:
As above.

Reply:
Fixed in 12af8b42f3a7e141d572c1cbe1cecf5ab38a067a.